docs/docs/start/self-hosting.mdx

```diff
@@ -43,7 +43,7 @@ docker build \
 
 ## AWS Elastic Beanstalk (Coming soon)
 
-We are working on providing a joint Docker image - containing both the Twenty frontend and server - that you can deploy using [AWS Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/).
+A joint Docker image - containing both the Twenty frontend and server - that you can deploy using [AWS Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/) is in the works. 
```

Comment:
we can also remove Twenty here (it was already there before but since I see it!)